openedx/adg/common/mailchimp_pipeline/__init__.py

```diff
@@ -1,4 +1,8 @@
 """
-Mailchimp pipeline app
+This app contains Mailchimp client and related tasks to send emails
```

Comment:
We don't send emails with mandrill, actually these are the tasks to sync data with mailchimp. Mandrill is used for sending emails